server/src/main/java/io/pravega/schemaregistry/storage/client/WireCommandClient.java

```diff
@@ -150,6 +150,7 @@
     CompletableFuture<List<TableSegmentKeyVersion>> updateTableEntries(final String tableName,
                                                                        final List<TableSegmentEntry> entries,
                                                                        String delegationToken) {
+        log.info("Update table entries wirecommand is called for tableName {}", tableName);
```

Comment:
Please change log level  to `trace`. You can refer to similar Controller code to understand how to assign log levels to messages. Here is another issue that can help : https://pravega-io.slack.com/archives/C02G6C290SW/p1633677468011600

Reply:
updated the same